android/anc/src/main/java/org/smartregister/fhircore/anc/data/model/PatientModels.kt

```diff
@@ -106,5 +106,37 @@ data class PatientVitalItem(
   var bpds: String = "",
   var bpdsUnit: String = "",
   var pulse: String = "",
-  var pulseUnit: String = ""
-)
+  var pulseUnit: String = "",
+  var bmi: String = "",
+  var bmiUnit: String = ""
+) {
+  fun isWeightAndHeightAreValid(): Boolean {
```

Comment:
this and below can be patterned `weightAndHeightAreValid` without `is`, or `isValidWeightAndHeight` to keep it grammatical, you can choose the variant you prefer

Reply:
updated